### ISSUE TITLE
extra pipe character introduce confusion

### ISSUE THREAD

**Issue by abednego4609** [poppinss/indicative#4]
accidental extra pipe character in schema declaration makes promises always rejected with empty (object/array) error objerct

``` javascript

var schema = {
    name: 'required|string|max:255|min:6|' /* <-- notice extra pipe here */

};
var v = {
    "name": "1234567"
};


indicative
    .validate(schema, v)
    .then(function (data) {
        // Never here.
        console.log('Hore! ' + JSON.stringify(data));
    })
    .catch(function (err) {
        // Always here!
        console.log('Nooo ' + JSON.stringify(err)); // Nooo {}
    });
```

**Comment by thetutlage** [comment]
Thanks for pointing these out , tests for parser itself are pending , so hope things would be getting fixed by weekend

**Comment by thetutlage** [comment]
Fixed and published to npm as indicative@1.0.5
